src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/visualmode/VisualModeMarkdownWriter.java

```diff
@@ -121,6 +122,7 @@ public Options optionsFromConfig(PanmirrorPandocFormatConfig formatConfig)
                {
                   referencesLocationPref = quarto.project_editor.markdown.references.location;
                   referencesPrefixPref = quarto.project_editor.markdown.references.prefix;
+                  referencesLinksPref = quarto.project_editor.markdown.references.links;
```

Comment:
@jjallaire Everything looked good except this one part. `quarto.project_editor.markdown.references.links` is an array of length 1. If I set the project level value to `false`, the reference links are still created.

I'm not sure if this was fixed in a later Quarto release if it's parsing the yaml there. I'm using 1.2.335.

Reply:
Okay, I'll take a closer look at this (I now have a local RStudio dev config so can actually run!)

Reply:
I just needed to add this bit to make sure `links` was read as a scalar: https://github.com/rstudio/rstudio/pull/12689/commits/d8a5aca44251cf2eb40893603b9b3d0f1fe282fe